package.json

```diff
@@ -14,7 +14,8 @@
     "start": "ember server",
     "check-style": "eslint .",
     "test": "yarn run check-style && ember test",
-    "test:cover": "COVERAGE=true yarn test"
+    "test:cover": "COVERAGE=true yarn test",
+    "postinstall": "npm rebuild node-sass"
```

Comment:
I've yet to see this actually fix anything once npm_modules is removed and reinstalled with up to date packages.
Why do you think it's necessary?